Lib/_pydatetime.py

```diff
@@ -284,6 +289,16 @@ def _wrap_strftime(object, format, timetuple):
                         push('{:04}'.format(year))
                         if ch == 'F':
                             push('-{:02}-{:02}'.format(*timetuple[1:3]))
+                elif ch == '-':
+                    if i < n:
+                        next_ch = format[i]
+                        i += 1
+                        if sys.platform.startswith('win') or sys.platform.startswith('android'):
```

Comment:
As the sys.platform docs explain, the `startswith` idiom is no longer required for any supported platform except FreeBSD. 

```suggestion
                        if sys.platform in ['win32', 'android']:
```